src/devices/map2DServer/Map2DServer.cpp

```diff
@@ -714,6 +714,12 @@ void Map2DServer::parse_string_command(yarp::os::Bottle& in, yarp::os::Bottle& o
         m_maps_storage.clear();
         out.addString("all maps cleared");
     }
+    else if (in.get(0).asString() == "enable_maps_compression")
+    {
+        for (auto it=m_maps_storage.begin(); it!= m_maps_storage.end(); it++)
+            {it->second.enable_map_compression_over_network(in.get(1).asBool());}
+        out.addString("compression mode of all maps set to:"+ in.get(1).asString());
```

Comment:
Please add a warning when compression is not enabled

Reply:
warning added